### PR TITLE
Add minimum Spring Boot version to docs.

### PR DESCRIPTION
**Pull request by maciejwalkowiak**
To avoid issues like https://github.com/getsentry/sentry-java/issues/970 this change makes it clear what's the minimum supported Spring Boot version.

### CODE REVIEW

File: src/platforms/java/guides/spring-boot/index.mdx [hunk 1]
```diff
@@ -8,6 +8,8 @@ The `sentry-spring-boot-starter` library enhances [Sentry Spring](/platforms/jav
 - automatically setting the release on `SentryEvent` when [Spring Boot Git integration is configured](https://docs.spring.io/spring-boot/docs/current/reference/html/howto.html#howto-git-info)
 - automatically registering `BeforeSendCallback`, `BeforeBreadcrumbCallback`, `EventProcessor`, `Integration` beans
 
+We support Spring Boot 2.1.0 or newer.
```
Comment (bruno-garcia) [inline src/platforms/java/guides/spring-boot/index.mdx:11]:
```suggestion
<Note><markdown>

We support Spring Boot 2.1.0 or newer. Still on an older version? You can use [our previous version](/platforms/java/legacy/spring/).

</markdown></Note>
```

**Review by bruno-garcia — APPROVED** [review]

**Comment by marandaneto** [comment]
https://github.com/getsentry/sentry-docs/pull/2481#issuecomment-705399688

**Review by marandaneto — APPROVED** [review]
LGTM

File: src/platforms/java/guides/spring-boot/index.mdx [hunk 2]
```diff
@@ -8,6 +8,12 @@ The `sentry-spring-boot-starter` library enhances [Sentry Spring](/platforms/jav
 - automatically setting the release on `SentryEvent` when [Spring Boot Git integration is configured](https://docs.spring.io/spring-boot/docs/current/reference/html/howto.html#howto-git-info)
 - automatically registering `BeforeSendCallback`, `BeforeBreadcrumbCallback`, `EventProcessor`, `Integration` beans
 
+<Note><markdown>
+
+We support Spring Boot 2.1.0 or newer. Still on an older version? You can use [our legacy SDK](/platforms/java/legacy/spring/).
```
Comment (PeloWriter) [inline src/platforms/java/guides/spring-boot/index.mdx:13]:
```suggestion
We support Spring Boot 2.1.0 and above. Still on an older version? You can use [our legacy SDK](/platforms/java/legacy/spring/).
```

**Review by PeloWriter — APPROVED** [review]
Thank you

**Pull request merged**